tools/nl/embeddings/build_embeddings.py

```diff
@@ -54,6 +64,8 @@
 flags.DEFINE_string('alternatives_filepattern', 'data/alternatives/main/*.csv',
                     'File pattern (relative) for CSVs with alternatives')
 
+flags.DEFINE_bool('dry_run', False, 'Dry run')
```

Comment:
Curious why have a separate flag for dry_run because it looks like dry run is only and always used for vertex ai?

Reply:
Right now vertex ai model only write to local files (then I manually upload to gcs and create index endpoint). I added a todo to use the dry_run for vertex ai mode as well.